boards/riscv/hifive1/hifive1_defconfig

```diff
@@ -10,7 +10,8 @@ CONFIG_UART_SIFIVE_PORT_0=y
 CONFIG_UART_CONSOLE=y
 CONFIG_PLIC=y
 CONFIG_PINMUX=y
-CONFIG_PINMUX_SIFIVE=y
+CONFIG_PINCTRL=y
+CONFIG_PINCTRL_SIFIVE=y
```

Comment:
only `CONFIG_PINMUX=y` should be required, see my previous comment about defaulting based on dt_compat_status. If all boards need this flag, consider enabling this at SoC level.

Reply:
Removed `CONFIG_PINCTRL_SIFIVE=y`